common/app/model/content.scala

```diff
@@ -687,17 +694,11 @@ object Gallery {
     val contentOverrides = content.copy(
       metadata = metadata,
       trail = trail,
-      rawOpenGraphImage = {
-        val bestOpenGraphImage = if (FacebookShareUseTrailPicFirstSwitch.isSwitchedOn) {
-          trail.trailPicture.flatMap(_.largestImageUrl)
-        } else {
-          None
-        }
```

Comment:
I have seen this `Boolean -> Option` pattern a lot before and it always looks like it can be simplified to me.  So I created the `def toOption` - let me know what you think.